docs/essentials/controls.md

```diff
@@ -79,25 +169,55 @@ By default, Storybook will choose a control for each arg based on its initial va
 
 </IfRenderer>
 
-<IfRenderer renderer={['ember', 'html', 'svelte', 'preact', 'qwik', 'solid', 'web-components' ]}>
+<IfRenderer renderer='web-components'>
+
+By default, Storybook will try to infer the required argTypes and associated controls for your stories based on the component's definition and the initial value of the args. You'll need to take some additional steps to set it up properly. You can opt to generate a [`custom-elements.json`](https://github.com/webcomponents/custom-elements-json) file with [`@custom-elements-manifest/analyzer`](https://github.com/open-wc/custom-elements-manifest) if you're using the `pre-v1.0.0` version of the elements file or [`@custom-elements-manifest/analyzer`](https://github.com/open-wc/custom-elements-manifest) for newer versions and configure it in your Storybook UI configuration file (i.e., `.storybook/preview.js|ts`) to enable it.
```

Comment:
Should the 2nd and 3rd links here be different?

Reply:
The links were based on a outdated version.